ui/DesignSystem/Component/SidebarLayout.svelte

```diff
@@ -15,9 +15,8 @@
   .container {
     position: relative;
     left: var(--sidebar-width);
-    width: calc(100vw - var(--sidebar-width));
+    min-width: calc(100vw - var(--sidebar-width));
     height: 100%;
-    min-width: 60rem;
```

Comment:
As described in the commit introducing this change, this was something discussed with Brandon. The app is not designed nor engineered, atm, to be responsive, and this change ensures that 1) the layout by definition always takes the whole viewport and 2) that if the content of the screen requires more space, that it should become scrollable.

I am not sure why - considering the size of your viewport - it is becoming scrollable for you since it has enough space to not go that route, I am not seeing that myself but I will have a look asaic.

Reply:
> I am not sure why - considering the size of your viewport - it is becoming scrollable for you since it has enough space to not go that route, I am not seeing that myself but I will have a look asaic.

I only raised this because there's enough space on my viewport (15" mbp) to display everything without needing a horizontal scrollbar. So you'll probably have to look into solution 2.

Reply:
>  Screen.Recording.2021-01-15.at.09.52.15.mov

It's not clear to me why this is happening. The support panels just have a min-width that is too large right? The min width doesn't need to be this big, as you can see, none of the UI is colliding with itself yet.

It appears to me we have this much space (see red arrow) left to continue condensing the UI. So why does the min-width stop here?
![image](https://user-images.githubusercontent.com/4406983/104898509-059d2900-597a-11eb-8e8b-c6838c5d6946.png)

Reply:
On my machine, I only get the scrolling when the components no longer fit inline, so I am not sure either why this is happening. @rudolfs (on mac) realized that if we resize the window back and forth that the issue disappears. I'm having a look.

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/975/commits/e3d30854855b53ddfdcd1c120a786389d46123ec